src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -138,25 +140,23 @@ private static void usage() {
   }
 
   public static void main(final String[] args) {
-    setupLogging();
-    ErrorConsole.getConsole().displayStandardError();
-    ErrorConsole.getConsole().displayStandardOutput();
-    ErrorHandler.registerExceptionHandler();
-    System.setProperty("triplea.engine.version", ClientContext.engineVersion().toString());
-    handleCommandLineArgs(args);
-    // do after we handle command line args
-    checkForMemoryXMX();
-    setupLookAndFeel();
-
-    LocalSystemChecker systemCheck = new LocalSystemChecker();
```

Comment:
Bad merge conflict resolution, good catch. The system check lines have been restored.